packages/devtools_app/lib/src/debugger/debugger_screen.dart

```diff
@@ -370,3 +370,73 @@ class _DebuggerStatusState extends State<DebuggerStatus> with AutoDisposeMixin {
     return 'paused$reason$fileName $pos';
   }
 }
+
+class FloatingDebuggerControls extends StatelessWidget {
+  @visibleForTesting
+  static const debuggerControlsKey = Key('debugger controls');
```

Comment:
general nit: why not find the buttons by their text instead of adding keys that are only used for testing?

Reply:
done.